apps/yaScrcpy GUI/uninstall

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -euo pipefail
+
+# --- Configuration (must match the install script) ---
+readonly APP_NAME="yascrcpy"
+readonly BIN_PATH="/usr/local/bin/${APP_NAME}"
+readonly ICON_PATH="/usr/share/pixmaps/${APP_NAME}.png"
+readonly DESKTOP_FILE="/usr/share/applications/${APP_NAME}.desktop"
+
+# --- Helper Functions (from Pi-Apps) ---
+# status() { ... }
+
+# --- Main Execution ---
+main() {
+    status "Starting uninstallation of ${APP_NAME}..."
+
+    status "Removing application binary..."
+    sudo rm -f "${BIN_PATH}"
+
+    status "Removing application icon..."
+    sudo rm -f "${ICON_PATH}"
+
+    status "Removing application shortcut..."
+    sudo rm -f "${DESKTOP_FILE}"
+
+    status "Updating icon cache..."
+    # Use || true to prevent the script from failing on non-critical cache update errors
```

Comment:
`|| true` is only necessary if the script has `set -e` enabled, causing the script to exit if any comnmands exit with a nonzero exit code. Pi-Apps does not use `set -e`, meaning it does not care about nonzero exit codes, making `|| true` unnecessary where failing is OK, but `|| error` very important where failing is not OK.